package.json

```diff
@@ -88,13 +88,14 @@
     "benchmark"
   ],
   "resolutions": {
-    "browserslist": "npm:4.21.5",
-    "caniuse-lite": "npm:1.0.30001460",
-    "core-js-compat": "npm:3.29.0",
-    "electron-to-chromium": "npm:1.4.320",
+    "browserslist": "npm:4.21.9",
+    "caniuse-lite": "npm:1.0.30001508",
+    "core-js-compat": "npm:3.31.0",
+    "electron-to-chromium": "npm:1.4.441",
     "glob-watcher/chokidar": "npm:^3.4.0",
     "@types/babel__core": "link:./nope",
     "@types/babel__traverse": "link:./nope",
+    "@babel/helper-define-polyfill-provider/@babel/helper-compilation-targets": "workspace:*",
```

Comment:
This is kind of deadlock: If we don't map compilation targets to the workspace one, the one used by `@babel/helper-define-polyfill-provider` will throw on unrecognized targets `opera_mobile` on tests. If we map the library, the build dependency `@babel/preset-env@npm` will be affected. 

One solution is to move `babel-polyfills` to the monorepo. I am investigating how to specify the resolutions so that the build deps are not affected.